python/ray/rllib/a3c/a3c.py

```diff
@@ -66,6 +66,7 @@
 class A3CAgent(Agent):
     _agent_name = "A3C"
     _default_config = DEFAULT_CONFIG
+    _allow_unknown_configs = True
```

Comment:
probably want to keep this as `False`

Reply:
without this I got `Unknown config parameter vf_loss_coeff`

Reply:
really? `vf_loss_coeff` is defined above in line 40; can you help me reproduce this error?

Reply:
Nevermind, I guess I mixed incompatible installation and `train.py` revisions.